core-it-suite/src/test/resources/mng-6759-transitive-dependency-repositories/module2/pom.xml

```diff
@@ -11,7 +11,7 @@
     <repositories>
         <repository>
             <id>customRepo</id>
-            <url>${custom.repo.uri}</url>
+            <url>file:///${basedir}/../dependency-in-custom-repo/target/repo</url>
```

Comment:
Isn't this deprecated for `{project.basedir}`?

Reply:
According to https://maven.apache.org/ref/current/maven-core/apidocs/org/apache/maven/plugin/PluginParameterExpressionEvaluator.html it is still valid

Reply:
My bad, it is indeed still valid...